README.md

```diff
@@ -8,7 +8,70 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running a demo on Windows
 
+Prerequisites:
+- Docker desktop
+
+[Download](https://github.com/ADAH-EviDENce/WorkingTitleCloseReader/archive/master.zip) a copy of WorkingTitleCloseReader archive and extract its contents on your machine.
+
+Open a Powershell prompt:
```

Comment:
Can we exaplain how we can start powershell?

Reply:
yeah why not, it could save our users a search
added it.